blackbox/cli.py

```diff
@@ -37,11 +38,15 @@ def run() -> bool:
 
     all_workflows = workflows.get_workflows(database_handlers, storage_handlers, notifier_handlers)
 
+    # Backups files we need to clean up
```

Comment:
```suggestion
    # Backup files we need to clean up
```

Reply:
Actually, this comment seems a bit redundant. I think the variable name `backup_files` already communicates what this contains when we remove them later.

I'd probably just get rid of this comment.

```suggestion
```